internal/handlers/shared/msg_collstats.go

```diff
@@ -30,7 +30,7 @@ func (h *Handler) MsgCollStats(ctx context.Context, msg *wire.OpMsg) (*wire.OpMs
 	}
 
 	m := document.Map()
-	collection := m[document.Command()].(string)
+	collection := m["collStats"].(string)
```

Comment:
Can it be invoked as `db.runCommand({cOlLsTaTs: "film"})`? Would it work?

Reply:
Yes, it can.

And MongoDB's response is:

```
monila> db.runCommand({ cOlLsTaTs: "actor" })
MongoServerError: no such command: 'cOlLsTaTs'
```

While FerretDB's response is:

```
monila> db.runCommand({ cOlLsTaTs: "actor" })
MongoNetworkError: connection 16 to 127.0.0.1:27017 closed
```

Reply:
Ok, so MongoDB command are case-sensitive